docs/development/Release-Procedure.md

```diff
@@ -26,6 +26,9 @@ We often run code in production that is not ready yet for use by the wider cBioP
     then put them in the right sections following same style as other releases: https://github.com/cBioPortal/cbioportal-frontend/releases.
 4. Create a news item with a link to your carefully crafted release notes. Highlight a few major changes that could be interesting to users of cBioPortal ideally with a screenshot similar to: https://github.com/cBioPortal/cbioportal/pull/6914/files?short\_path=6f95322#diff-6f953229832059bab3fe229d4af08b52 (in the files changed section, you can click on view rich diff to see the converted markdown).
 
+## Backend Hotfixes
+Occasionally there are bugs identified in the backend running in production. Creating an entire new release can be cumbersome so in this case, we create a new release branch instead. E.g. if the current release is `6.0.2`, the new branch would be `release-6.0.3`. Any fixes are then merged to this branch and if it looks good, they are deployed to production. On the following Tuesday whatever's in this `release-x` branch get merged to master and included in that Tuesday's release. 
```

Comment:
Can we have a meeting on this? I am not sure if this is exactly what we want to do.